README.md

```diff
@@ -1,47 +1,117 @@
-# OGC Features and Geometries JSON
+# OGC Features and Geometries JSON (JSON-FG)
 
 This GitHub repository contains [OGC](https://www.ogc.org/)'s candidate standard for OGC Features and Geometries JSON.
 
-OGC Features and Geometries JSON will build on the widely used [GeoJSON standard](https://geojson.org/) and extend it with minimal extensions to support additional concepts that are important for the wider geospatial community and the [OGC API standards](https://ogcapi.ogc.org/).
+OGC Features and Geometries JSON (JSON-FG) builds on the widely used [GeoJSON standard](https://geojson.org/) and extends it with minimal extensions to support additional concepts that are important for the wider geospatial community and the [OGC API standards](https://ogcapi.ogc.org/).
 
 ## Status
 
-This is a new OGC working group and the work has started on June 1st, 2021. A first draft of the candidate standard, version 0.1.0, is available.
+The current version is 0.2.2. 
+
+This version will be the basis for the OGC Public Comment phase for 90 days, which is the final call for comments before the candidate standard will go through the process for approval as an OGC Standard after resolving the comments received during the OGC Public Comment period.
 
 The editor's draft of the specification can be found at [docs.ogc.org/DRAFTS/21-045.html](https://docs.ogc.org/DRAFTS/21-045.html). A [PDF version](https://docs.ogc.org/DRAFTS/21-045.pdf) is available, too.
 
-Next steps:
+The Open Geospatial Consortium (OGC) invites organisations and developers that have a need for the extensions specified by this specification to implement and test the extensions. Please submit feedback in the [in this repository](https://github.com/opengeospatial/ogc-feat-geo-json/issues). Are these extensions useful for your use cases? Are they simple enough to implement?
```

Comment:
remove "in the" just before "[in this repository]

Reply:
removed "in the"